src/main/java/org/example/camunda/process/solution/service/FormService.java

```diff
@@ -21,6 +22,8 @@ public class FormService {
 
   public static final String FORMS = "forms";
 
+  @Autowired private BpmnUtils bpmnUtils;
```

Comment:
If we change BpmnUtils as a service we should perhaps rename it. Better speak about it before taking any decision.